_pages/en/strategy-learning-automating-improving.md

```diff
@@ -36,51 +35,51 @@ Strategies will be reviewed and updated quarterly based on current progress and
 - [Mandatory Procedures on APIs](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604)
 - [Acceptable Network and Device Use](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32605)
 
-### Effective Immediately - IT teams must
-
-- [Use 20% of time to learn, automate and improve](#use-20-of-time-to-learn-automate-and-improve)
-- [Have access to resources and modern IT tools](#have-access-to-resources-and-modern-it-tools)
+### IITB teams must
 
 #### Use 20% of time to learn, automate and improve
 
-##### Actions
-
-- **Make it a requirement in performance agreements of directors, managers and team leads**
-- **Use CATS codes for continuous improvement and learning**
-- **Invest in training on agile for managers and team leads**
-- **Encourage assignments with CDS-ESDC team or CSPS**
-- **Invest to establish agile/DevOps Dojos for IT teams**
+- **Use CATS codes for continuous improvement**
+- **Gather metrics for things your teams work on**
+  - **Deployment (change) frequency**
+  - **lead time for change (or delivery of service/device)**
+  - **Time to restore (following issue with change)**
+  - **Change fail rate**
 
 ##### Guidance
 
 - Read, take courses, attend events and conferences
 - Stay up to date on trends and technologies
+- Encourage assignments with CDS-ESDC team or CSPS
 - Share knowledge with colleagues
-  - IITB showcase, Dev CoP, ..
+  - IITB showcase, Dev CoP, GCconnex, GCcollab, ..
 - Review existing processes and governance structure
 - Automate where possible
-- Show progress on automation and continuous improvements
 
-#### Have access to resources and modern IT tools
+### IITB must
 
-##### Actions
+#### Before Q1
 
-- **Create cloud and desktop sandboxes (labs) for experimentation**
-- **[Unblock access to specific SaaS for IITB employees](web-services-access.html)**
-- **Organize regular informal meetings between senior management and a team/project that recently "failed"**
-- **Invest in Bandwidth, Data Loss Protection (DLP) and a Document Classification Tool to enable more open access to Internet**
-- **Add all approved desktop software, including commercial and OSS, to Application Catalogue**
+- Make quarterly strategies a requirement in performance agreements of directors, managers and team leads
+- Create cloud and desktop sandboxes (labs) for experimentation
+- [Unblock access to Web tools for IITB employees](web-services-access.html)
+- Add all approved desktop software, including commercial and OSS, to Application Catalogue
+- Something about experimentation/failure and management support..
```

Comment:
Maybe remove? Not sure this is fully tangible. Can also be second iteration of continuous Learning.